app/scripts/views/sign_in.js

```diff
@@ -95,6 +96,12 @@ define(function (require, exports, module) {
       this._formPrefill.set('password', this.getElementValue('.password'));
     },
 
+    hideSuccessAndErrorMessages: function () {
+      $(this.el).off('keypress', '.input-row');
```

Comment:
I don't think you need this here because we want to keep hiding new errors if they show up. Was there a reason you added this here?